modules/server/src/network/utils/sqon.ts

```diff
@@ -0,0 +1,6 @@
+import SQONBuilder from '@overture-stack/sqon-builder';
+
+export const isSQONFilter = (filter: unknown) => {
+	// An error will be thrown if the provided input is invalid.
+	SQONBuilder.from(filter);
+};
```

Comment:
Shouldn't this return boolean to check the data type? If you want an error thrown at a specific location then throw from there, but this won't be very reusable if it either does nothing or throws.

Also, lets add a TSDocs comment for this exported utility.

Reply:
the return value is a SQON object, which we don't want, we want to keep it as a string.
So the only purpose this has is to throw an error if it's not a SQON